src/handlers/issue-deduplication.ts

```diff
@@ -0,0 +1,138 @@
+import { IssueSimilaritySearchResult } from "../adapters/supabase/helpers/issues";
+import { Context } from "../types";
+const MATCH_THRESHOLD = 0.95;
+const WARNING_THRESHOLD = 0.5;
```

Comment:
Why did you do 50%?

Reply:
A cosine similarity of 0.75 appears quite close for identifying similar issues. I tested this with a few examples and noticed some potential errors with the samples. Typically, for similar issues, the similarity was either above 75% and aligned with 95% category or around 60%. Therefore, I experimented with a 50% threshold, which seemed to work well.